code/unclassified/average/average.scala

```diff
@@ -0,0 +1,9 @@
+object Average extends App{
+	def average(num:Array[Double]):Double = {
+		val sum:Double = num.foldLeft(0.0){(a, b) => a + b}
+		sum / num.size
+	}
+
+	val arr:Array[Double] = Array(1, 2, 3, 4, 5)
+	println(average(arr))
+}
```

Comment:
Please add a new line after the code ends ! 👍

Reply:
Any technical reason for this?

Reply:
Some compilers (especially C or C++ ones) give you warnings about:

"No new line at end of file"

Reply:
Scala is not C or C++. Moreover,  we do not see a new line at the end of file in your pull requests containing C code 👍

Reply:
@AdiChat It is required by the C and Pre C++11 standard that there is a newline at the end of every source file ([Source](https://stackoverflow.com/a/8172281/6525260))